pygeode/stats.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 from scipy.stats import norm, t as tdist
 
-def correlate(X, Y, axes=None, output = 'r2,p', pbar=None):
+def correlate(X, Y, axes=None, output = 'r,p', pbar=None):
 # {{{
   r'''Computes correlation between variables X and Y.
```

Comment:
While we're correcting my goofs, let's make the description more precise and call this Pearson correlation coefficient

Reply:
Sure.